appengine/cloudsql/README.md

```diff
@@ -0,0 +1,12 @@
+# Cloud SQL sample for Google App Enginge
+This sample demonstrates how to use [Cloud SQL](https://cloud.google.com/sql/) on Google App Engine
+## Setup
+Before you can run or deploy the sample, you will need to create a [Cloud SQL instance](https://cloud.google.com/sql/docs/create-instance)
+1. Create a new user and database for the application. The easiest way to do this is via the [Google Developers Console](https://console.developers.google.com/project/_/sql/instances/example-instance2/access-control/users). Alternatively, you can use MySQL tools such as the command line client or workbench.
```

Comment:
Can't I connect to Cloud SQL v2 instances when I'm local?  If so, why not tell them how?